python/lsst/ts/genericcamera/driver/alliedvisioncamera.py

```diff
@@ -81,6 +83,7 @@ def initialise(self, config):
         self.plate_scale = config.config["plate_scale"]
 
         with vimba.Vimba.get_instance() as v:
+            # v.enable_log(vimba.LOG_CONFIG_TRACE_CONSOLE_ONLY)
```

Comment:
Is it absolutely necessary to keep this commented out line? If not, please remove.

Reply:
Yes, I have occasional use for it and it's easier to leave it around then having to find the syntax in the library code and put it back in.